lib/credit_card_widget.dart

```diff
@@ -256,45 +324,80 @@ class _CreditCardWidgetState extends State<CreditCardWidget>
             child: _buildBackContainer(),
           ),
         ),
+        if (!FlutterCreditCardPlatform.instance.isGyroscopeAvailable &&
+            widget.shouldFloat)
+          Positioned.fill(
+            child: LayoutBuilder(
+              builder: (_, BoxConstraints constraints) {
+                final double parentHeight = constraints.maxHeight;
+                final double parentWidth = constraints.maxWidth;
+                final double outerPadding =
+                    (screenSize?.width ?? parentWidth) - parentWidth;
+                final double padding = outerPadding != 0 && widget.padding == 0
+                    ? AppConstants.creditCardPadding
+                    : widget.padding;
+
+                return CursorListener(
```

Comment:
I belive this is only used as listner so can we avoid this widget from build and use some other workaround?

Reply:
That is correct, but it listens over a particular region, which would be the credit card widget, and thus it is in the build method.